RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractPrefetch.java

```diff
@@ -0,0 +1,388 @@
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2020 Basis Technology Corp.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.recentactivity;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Level;
+import org.openide.modules.InstalledFileLocator;
+import org.openide.util.NbBundle.Messages;
+import org.sleuthkit.autopsy.casemodule.Case;
+import org.sleuthkit.autopsy.casemodule.services.FileManager;
+import org.sleuthkit.autopsy.coreutils.ExecUtil;
+import org.sleuthkit.autopsy.coreutils.Logger;
+import org.sleuthkit.autopsy.coreutils.PlatformUtil;
+import org.sleuthkit.autopsy.coreutils.SQLiteDBConnect;
+import org.sleuthkit.autopsy.datamodel.ContentUtils;
+import org.sleuthkit.autopsy.ingest.DataSourceIngestModuleProcessTerminator;
+import org.sleuthkit.autopsy.ingest.DataSourceIngestModuleProgress;
+import org.sleuthkit.autopsy.ingest.IngestJobContext;
+import org.sleuthkit.datamodel.AbstractFile;
+import org.sleuthkit.datamodel.Blackboard;
+import org.sleuthkit.datamodel.BlackboardArtifact;
+import static org.sleuthkit.datamodel.BlackboardArtifact.ARTIFACT_TYPE.TSK_ASSOCIATED_OBJECT;
+import org.sleuthkit.datamodel.BlackboardAttribute;
+import static org.sleuthkit.datamodel.BlackboardAttribute.ATTRIBUTE_TYPE.TSK_ASSOCIATED_ARTIFACT;
+import org.sleuthkit.datamodel.Content;
+import org.sleuthkit.datamodel.TskCoreException;
+
+/**
+ * Extract the Prefetch Files and process them thru an External program.  The data will then be added to the 
+ * TSK_PROG_RUN artifact.  Associated artifacts will be created if possible. 
+ */
+final class ExtractPrefetch extends Extract {
+
+    private static final Logger logger = Logger.getLogger(ExtractPrefetch.class.getName());
+
+    private IngestJobContext context;
+
+
+    private static final String MODULE_NAME = "extractPREFETCH"; //NON-NLS
+
+    private static final String PREFETCH_TSK_COMMENT = "Prefetch File";
+    private static final String PREFETCH_FILE_LOCATION = "/Windows/Prefetch";
+    private static final String PREFETCH_TOOL_FOLDER = "markmckinnon"; //NON-NLS
+    private static final String PREFETCH_TOOL_NAME_WINDOWS_64 = "parse_prefetch_64.exe"; //NON-NLS
```

Comment:
I completely missed this the first time around, but InstalledFileLocator problem was simply because these names don't match whats in thirdparty. They should be:

'parse_prefetch_x64.exe'
'parse_prefetch_x86.exe'

Reply:
Not sure what happened there but it is now fixed.